attractors_pick:  [ -0.10,  0.70, 0.25, -0.20,  0.65, 0.25, 0,  0.65, 0.25]
attractor_write:  [ -0.10,  0.40, 0.25]
